README.md

```diff
@@ -20,8 +20,12 @@ you can stick on the wall to impress your grandma.
 
 ### Install it
 
-- `npm install --save-dev dependency-cruiser` (with `yarn` or `pnpm` use their
-  equivalent to install & save dependency-cruiser as a development dependency).
+```javascript
```

Comment:
```suggestion
```shell
```

These commands call javascript, but they are run from the shell. Hence, usign ````shell` in stead gives better highlighting (the `-`'s don't look as syntax errors anymore :-) )